python/raft-ann-bench/src/raft-ann-bench/run/conf/algos/raft_ivf_pq.yaml

```diff
@@ -6,12 +6,24 @@ groups:
   base:
     build:
       nlist: [1024, 2048, 4096, 8192]
-      pq_dim: [64, 32]
+      pq_dim: [64, 32, 16, 8]
```

Comment:
Do we have datasets where tests with `pq_dim=8` is relevant?

Reply:
I don't think so. I was looking at the benchmarking results and for all parameter settings with `pq_dim = 8`, the recall is `< 0.6`.  As such, we can certainly remove `pq_dim = 8` from both -- base and large settings.